app/views/placements/schools/mentors/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= content_for :page_title, sanitize("#{@mentor.full_name} - #{@school.name}") %>
+
+<%= render "placements/schools/primary_navigation", school: @school, current_navigation: :mentors %>
+
+<%= content_for(:before_content) do %>
+  <%= govuk_back_link(href: placements_school_mentors_path(school_id: @school.id)) %>
+<% end %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l"><%= @school.name %></span>
+      <h2 class="govuk-heading-l"><%= @mentor.full_name %></h2>
+
+      <%= govuk_summary_list do |summary_list| %>
+        <% summary_list.with_row do |row| %>
+          <% row.with_key(text: t(".attributes.mentors.first_name")) %>
```

Comment:
for first and last names, you might want to use `Mentor.human_attribute_name.first_name` (or last_name)

I think we will probably render the trn field in multiple ways so good to use a defined translation for that one (so we don't end up with the little problem we have with user email at the moment. Someone has defined the human attributes as "Email address", but sometimes we just want "Email")

Reply:
resolved in [f66b7fb](https://github.com/DFE-Digital/itt-mentor-services/pull/274/commits/f66b7fbdbaca1bed6ce46733c6c8777cdf3e27aa)